src/main/java/org/mifos/connector/airtel/mockairtel/service/AirtelMockService.java

```diff
@@ -0,0 +1,154 @@
+package org.mifos.connector.airtel.mockairtel.service;
+
+import java.util.UUID;
+import org.mifos.connector.airtel.mockairtel.schema.TransactionStatus;
+import org.mifos.connector.airtel.mockairtel.schema.TransferStatus;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelEnquiryResponseDTO;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelEnquiryResponseDataDTO;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelEnquiryResponseDataTransactionDTO;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelPaymentRequestDTO;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelPaymentResponseDTO;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelPaymentResponseDataDTO;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelPaymentResponseDataTransactionDTO;
+import org.mifos.connector.common.mobilemoney.airtel.dto.AirtelResponseStatusDTO;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AirtelMockService {
+
+    @Value("${mock-airtel.MSISDN_SUCCESSFUL}")
+    private String msisdnSuccessful;
+
+    @Value("${mock-airtel.MSISDN_FAILED}")
+    private String msisdnFailed;
+
+    @Value("${mock-airtel.TRANSACTION_ID_SUCCESSFUL}")
+    private String transactionIdSuccessful;
+
+    @Value("${mock-airtel.TRANSACTION_ID_FAILED}")
+    private String transactionIdFailed;
+
+    @Value("${mock-airtel.SUCCESS_RESPONSE_CODE}")
+    private String successResponseCode;
+
+    @Value("${mock-airtel.FAILED_RESPONSE_CODE}")
+    private String failedResponseCode;
+
+    @Value("${mock-airtel.PENDING_RESPONSE_CODE}")
+    private String pendingResponseCode;
+
+    @Value("${mock-airtel.RESULT_CODE}")
+    private String resultCode;
+
+    public ResponseEntity<AirtelEnquiryResponseDTO> getTransactionStatus(String transactionId) {
+        String airtelMoneyId = UUID.randomUUID().toString().replace("-", "");
+        String message;
+        String status;
+        String code;
+        String responseCode;
+        boolean success;
+        HttpStatus httpStatus;
+
+        if (transactionId.equals(transactionIdSuccessful)) {
+            message = TransferStatus.SUCCESSFUL.name();
+            status = TransactionStatus.TS.name();
+            code = HttpStatus.OK.toString();
```

Comment:
The code now has the value 200 Ok instead of 200 is that fine? Otherwise I think we have to create a seperate enum for response codes.